mbuild/compound.py

```diff
@@ -80,6 +83,35 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
              "distances and standard residue templates!")
         structure = pmd.load_file(filename, structure=True, **kwargs)
         compound.from_parmed(structure, coords_only=coords_only)
+
+    elif smiles:
+        pybel = import_('pybel')
+        # First we try treating filename as a SMILES string
+        try:
+            mymol = pybel.readstring("smi", filename)
+        # Now we treat it as a filename
+        except(OSError, IOError):
+            # For now, we only support reading in a single smiles molecule,
+            # but pybel returns a generator, so we get the first molecule
+            # and warn the user if there is more
+
+            mymol_generator = pybel.readfile("smi", filename)
+            mymol_list = list(mymol_generator)
+            if len(mymol_list) == 1:
+                mymol = mymol_list[0]
+            else:
+                # Warn
+                warn("More than one SMILES string in file, more than one SMILES "
+                     "string is not supported, using the first")
```

Comment:
Can we print out the SMILES string that is being used in this warning message? Would help make it a bit less ambiguous.

Reply:
Realized I marked this only as a comment. Pinging @mikemhenry just in case this didnt raise a notification.

Reply:
Good idea!